src/components/Login/Login.tsx

```diff
@@ -0,0 +1,50 @@
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import {
+  Box,
+  Checkbox,
+  Typography,
+  Link,
+  Stack,
+  Button,
+} from '@mui/material/';
+
+import CustomTextFieldGroup from '../CustomTextFieldGroup/CustomTextFieldGroup';
+import { LoginTextFields } from './LoginTextFieldsData';
+
+const preventDefault = (event: React.SyntheticEvent) =>
+  event.preventDefault();
+
+const Login = () => {
+  return (
+    <Box
+      component="form"
+      sx={{ '& > :not(style)': { m: 1, width: '35ch' } }}
+      noValidate
+      autoComplete="off"
+      onClick={preventDefault}
```

Comment:
interesting, I always do:

```suggestion
      onClick={() => undefined}
```

😅

Reply:
obviously I do this for WIP changes